modules/btrfs/data/org.freedesktop.UDisks2.btrfs.xml

```diff
@@ -219,6 +219,20 @@
       <arg name="options" type="a{sv}" direction="in"/>
     </method>
 
+    <!--
+        GetDefaultSubvolumeID:
+        @options: Additional options.
+        @since: 2.11.0
```

Comment:
FYI: not sure if this is correct.

Reply:
The version number? Yes, that's correct.